### DISCUSSION TITLE
nsclient++ log

### DISCUSSION THREAD

**Discussion by alacran** [mickem/nscp#787] [General]
Hello, where do I find the nsclient log on the monitored server? Or where do the events of this program appear?

**Comment by jadzdotcom** [comment]
The FAQ (https://nsclient.org/docs/faq/) has an entry on "How do I enable debug log" which mentions the required nsclient.ini settings. 

You can also run the client in 'test' mode and see output from the service directly (i.e. shut down service and then run `nscp test` manually to view the output on the console).

**Reply by alacran** [reply]
Thanks. I want to record when the nsclient service is started or stopped in the file, but I made a new one and unchecked this the .ini. It doesn´t recorder nothing of nothing. what would i be missing?
![restart_nsclient](https://user-images.githubusercontent.com/5720958/210882057-6e69212c-5995-41c7-afae-7402a0cce269.PNG)

![test2](https://user-images.githubusercontent.com/5720958/210881526-12ef449a-c649-472c-8e01-de07cfa3d270.PNG)
![ten1](https://user-images.githubusercontent.com/5720958/210881529-c123482b-3480-499e-b46d-fc337757def1.PNG)

**Comment by jadzdotcom** [comment]
Are you running a very old version of the software? We might need to look at the documentation for that older version.

**Reply by alacran** [reply]
thanks, i found nsclient errors in event viewer. Application error.
![sndowse3_20221228](https://user-images.githubusercontent.com/5720958/211070201-5acbb774-bcb8-4022-b268-7bbd67fb63a6.PNG)
![error_application_nsclient_20221228](https://user-images.githubusercontent.com/5720958/211070206-029efba0-94ea-47b2-b098-27db6da606b4.PNG)
![sndpe2_20230301_1231_view](https://user-images.githubusercontent.com/5720958/211070208-c1c97621-9b58-4601-8bbb-b5e7766f219e.PNG)
![sndpe2_20230301_1231](https://user-images.githubusercontent.com/5720958/211070212-ccfa6e2e-2402-4d97-9b87-0088ad876c31.PNG)
